### DISCUSSION TITLE
Update 1.15.7.0

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4433] [General]
Hi all,

after the new update from MS, I saw a first funny behaviour of the custom FBW mod. When you go via ESC to the menu and then resume fling, the flaps lever is on FLAPS 2 and the spoiler lever is also in 1/2 position ... :-p ..... 

good night 

Klaus

**Comment by Annolago** [comment]
That has been happening to me for a long time, however it has always been flaps full. So whenever going back after having ESC to the menu functions while in flight would trigger master warning due to overspeed as the flaps, as mentioned before, are in flaps full config. 
Should note that it also happens in "non flight" situations - read while on the ground - but naturally without the overspeed warning.

**Reply by ChruutvoLuzi** [reply]
thats a different thing and is not really an issue. You have to unbind all non existing axises

**Reply by CptBayer** [reply]
ah ok, means what? All axis are in use in my simulator? hmmm .....

greets klaus

**Reply by ChruutvoLuzi** [reply]
sorry, that was directed to Annalago. I have the same issue as you describe with the 50% values when hiting esc. Seems to be a issue lots of people have
